debian/control

```diff
@@ -17,12 +17,17 @@ Package: rednotebook
 Architecture: all
 Depends: ${python3:Depends},
          ${misc:Depends},
+         gir1.2-gdk-3.0,
          gir1.2-gdkpixbuf-2.0,
+         gir1.2-gio-2.0,
+         gir1.2-girepository-2.0,
          gir1.2-glib-2.0,
+         gir1.2-gobject-2.0,
          gir1.2-gtk-3.0,
          gir1.2-gtksource-4,
          gir1.2-pango-1.0,
          gir1.2-webkit2-4.1 | gir1.2-webkit2-4.0,
+         gir1.2-webkit2-4.1,
```

Comment:
Did this one slip in by accident? If not, we should remove the line above it, I guess.

Reply:
That is a Phil mess up. The line is that in Debian unstable/sid and testing/trixie, because the Debian releases this will be backported to have 4.1, so the 4.0 is now not necessary.

Reply:
Dependant on how far you wish to backport, the 4.0 can be removed or stay. :-)

Reply:
Then let's keep it to preserve support for Ubuntu 20.04.